### ISSUE TITLE
Add rdfs:isDefinedBy for every ontology term during the release bundle operation

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#265]
rdfs:isDefinedBy should point to the ontology defined in the file.

This will be handled by adding a new operation to ontology-toolkit ([PR 25](https://github.com/semanticarts/ontology-toolkit/pull/25), [PR 26](https://github.com/semanticarts/ontology-toolkit/pull/26) and an update to the gist bundle.yaml file (PR #264).

**Comment by sa-bpelakh** [comment]
Closed by https://github.com/semanticarts/gist/pull/264, but will be OBE by removal of modules.

**Comment by uscholdm** [comment]
@sa-bpelakh said:
> Closed by #264, but will be OBE by removal of modules.

I disagree. Many ontologies will import gistCore. Every concept should have an isDefinedBy axiom to make that clear.

**Comment by sa-bpelakh** [comment]
As I said, it is covered by https://github.com/semanticarts/gist/pull/264, so we do have this taken care of.

**Comment by uscholdm** [comment]
@sa-bpelakh  said:
> As I said, it is covered by #264, so we do have this taken care of.

Issue https://github.com/semanticarts/gist/pull/264 says:
> rjyounes deleted the bundle-defined-by branch on May 15

Which makes me think that the `isDefinedBy` work is not included now. 
What I objected to is this:
> Closed by #264, but will be OBE by removal of modules.

which seem to be saying that with gist modules gone, we will not need to add isDefinedBy triples and include that functionality in the bundling.  If this is indeed now done, that's fine -- but I found the verbiage misleading.
